src/Approvals/SFA.DAS.Approvals.UnitTests/Application/Cohorts/Commands/CreateCohortCommandHandlerTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using AutoFixture;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using SFA.DAS.SharedOuterApi.Models;
+using System.Net;
+using System.Threading.Tasks;
+using System.Threading;
+using FluentAssertions;
+using SFA.DAS.Approvals.Application.Cohorts.Commands.CreateCohort;
+using SFA.DAS.Approvals.InnerApi.Requests;
+using SFA.DAS.Approvals.InnerApi.Responses;
+
+namespace SFA.DAS.Approvals.UnitTests.Application.Cohorts.Commands
+{
+    [TestFixture]
+    public class CreateCohortCommandHandlerTests
+    {
+        private CreateCohortCommandHandler _handler;
+        private CreateCohortCommand _request;
+        private Mock<ICommitmentsV2ApiClient<CommitmentsV2ApiConfiguration>> _commitmentsApiClient;
+        private Fixture _fixture;
+
+        [SetUp]
+        public void Setup()
+        {
+            _fixture = new Fixture();
+            _request = _fixture.Create<CreateCohortCommand>();
+
+            _commitmentsApiClient = new Mock<ICommitmentsV2ApiClient<CommitmentsV2ApiConfiguration>>();
+
+            _handler = new CreateCohortCommandHandler(_commitmentsApiClient.Object);
+        }
+
```

Comment:
Do we want to add some non-happy path tests in here in the case that the response from commitments is null (ArgumentNullException thrown) or unsuccessful (ApiResponseException thrown)?